src/main/java/ticketservice/dao/UserDao.java

```diff
@@ -1,37 +1,38 @@
-package dao;
+package ticketservice.dao;
 
-import jdbc.CustomDataSource;
-import user.User;
+import org.springframework.stereotype.Component;
+import ticketservice.user.User;
 
+import javax.sql.DataSource;
 import java.sql.*;
 
+@Component
```

Comment:
for DAO level it is better to use specific annotation: `@Repository`

Reply:
Fixed